README.md

```diff
@@ -1,134 +1,156 @@
+  * **routes-segments-snake-case** - Route's dynamic segments should use snake case [(more)](https://github.com/netguru/eslint-plugin-ember/#route-naming)
 
 \* Rule with optional settings
 
 ### All Rules in JSON
 
 ```
-    "netguru-ember/alias-model-in-controller": 0,
-    "netguru-ember/avoid-leaking-state-in-components": 0,
-    "netguru-ember/closure-actions": 0,
-    "netguru-ember/jquery-ember-run": 0,
-    "netguru-ember/local-modules": 0,
-    "netguru-ember/named-functions-in-promises": 0,
-    "netguru-ember/no-empty-attrs": 0,
-    "netguru-ember/no-function-prototype-extensions": 0,
-    "netguru-ember/no-observers": 0,
-    "netguru-ember/no-on-calls-in-components": 0,
-    "netguru-ember/no-side-effects": 0,
-    "netguru-ember/order-in-components": 0,
-    "netguru-ember/order-in-controllers": 0,
-    "netguru-ember/order-in-models": 0,
-    "netguru-ember/order-in-routes": 0,
-    "netguru-ember/query-params-on-top": 0,
-    "netguru-ember/routes-segments-snake-case": 0,
-    "netguru-ember/use-brace-expansion": 0,
-    "netguru-ember/use-ember-get-and-set": 0,
+    "ember/alias-model-in-controller": 0,
+    "ember/avoid-leaking-state-in-components": 0,
+    "ember/closure-actions": 0,
+    "ember/jquery-ember-run": 0,
+    "ember/local-modules": 0,
+    "ember/named-functions-in-promises": 0,
+    "ember/no-empty-attrs": 0,
+    "ember/no-function-prototype-extensions": 0,
+    "ember/no-observers": 0,
+    "ember/no-on-calls-in-components": 0,
+    "ember/no-side-effects": 0,
+    "ember/order-in-components": 0,
+    "ember/order-in-controllers": 0,
+    "ember/order-in-models": 0,
+    "ember/order-in-routes": 0,
+    "ember/query-params-on-top": 0,
+    "ember/routes-segments-snake-case": 0,
+    "ember/use-brace-expansion": 0,
+    "ember/use-ember-get-and-set": 0,
 ```
 
 ## Contribution guide
 
-TBD. For now feel free to add any suggestions in [issues](https://github.com/netguru/eslint-plugin-netguru-ember/issues). Any involvement highly appreciated.
+In order to add new rule, you should:
+- create `rules/rule-name.js` file
+- create `test/rule-name.js` file
+- describe new rule in [docs/RULES.md](https://github.com/netguru/eslint-plugin-ember/blob/master/docs/RULES.md) (remember about setting proper anchor)
+- add new rule in [README.md](https://github.com/netguru/eslint-plugin-ember/blob/master/README.md#rules)
+
+Recommended way of creating new rule:
+- Create issue on GH with description of proposed rule
+- Write sufficient test scenarios
+- Run `gulp watch`
+- Implement logic for new rule
+- Make sure all tests passes
+- Add documentation and update README
+- Create PR and link created issue in description
+
+If you have any suggestions, ideas or problems feel free to add new [issue](https://github.com/netguru/eslint-plugin-ember/issues, but first please make sure your question does not repeat previous ones.
+
+## Author
+- [Michał Sajnóg](http://github.com/michalsnik)
```

Comment:
I just wondered how we'll decide who should be in this section and who not. Should we add to contributors in README anyone that created PR even if it was a typo fix in README?